docs/API-DESIGN.md

```diff
@@ -0,0 +1,881 @@
+  // Allows to specify a list of tables that should not be repaired. Cannot be used in conjunction with Tables.
+  IgnoredTables []string `url:"blacklistedTables,comma,omitempty"`
+
+  // Defines the amount of segments per node to create for the repair run. The value must be >0 and <=1000.
+  SegmentCountPerNode int `url:"segmentCountPerNode,omitempty"`
+
+  // Defines the used repair parallelism for repair run.
+  RepairParallelism RepairParallelism `url:"repairParallelism,omitempty"`
+
+  // Defines the used repair parallelism for repair run.
+  Intensity Intensity `url:"intensity,omitempty"`
+
+  // Defines if incremental repair should be done.
+  IncrementalRepair bool `url:"incrementalRepair,omitempty"`
+
+  // When to trigger the next repair run. If not specified, defaults to the next day, at start of day.
+  TriggerTime *time.Time `url:"scheduleTriggerTime,omitempty"`
+
+  // Allows to specify a list of nodes whose tokens should be repaired.
+  Nodes []string `url:"nodes,comma,omitempty"`
+
+  // Allows to specify a list of datacenters to repair.
+  Datacenters []string `url:"datacenters,comma,omitempty"`
+
+  // Defines the thread count to use for repair. Since Cassandra 2.2, repairs can be performed with
+  // up to 4 threads in order to parallelize the work on different token ranges.
+  RepairThreadCount int `url:"repairThreadCount,omitempty"`
+}
+
+type PercentRepairedMetric struct {
+  Cluster          string    
+  Node             string    
+  RepairScheduleId uuid.UUID 
+  Keyspace         string    
+  Table            string    
+  PercentRepaired  int       
+}
+
+type RepairScheduleState string
+
+const (
+  RepairScheduleStateActive  = RepairScheduleState("ACTIVE")
+  RepairScheduleStatePaused  = RepairScheduleState("PAUSED")
+  RepairScheduleStateDeleted = RepairScheduleState("DELETED")
+)
+```
+
+## Snapshot Resource
+
+
+### Methods
+
+Methods in this resource come in pairs: one for a specific node, one for a cluster-wide equivalent. They are distinguished by "[VERB]NodeSnapshot"vs "[VERB]ClusterSnapshot".
+
+
+<table>
+  <tr>
+   <td><strong>REST endpoint</strong>
+   </td>
+   <td><strong>Client API method</strong>
+   </td>
+   <td><strong>Comments</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><code>GET /snapshot/{clusterName}/{host}</code>
+   </td>
+   <td><code>NodeSnapshots(ctx context.Context, cluster string, node string) ([]*Snapshot, error)</code>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td><code>GET /snapshot/cluster/{clusterName}</code>
+   </td>
+   <td><code>ClusterSnapshots(ctx context.Context, cluster string) ([]*Snapshot, error)</code>
+   </td>
+   <td> 
+   </td>
+  </tr>
+  <tr>
+   <td><code>POST /snapshot/{clusterName}/{node}</code>
+   </td>
+   <td><code>CreateNodeSnapshot(ctx context.Context, cluster string, node string, options *NodeSnapshotCreateOptions) (string, error)</code>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td><code><em>POST /snapshot/cluster/{clusterName}</em></code>
```

Comment:
Do you want <em> tag on this one?

Reply:
Nice catch, I think the tag was generated by the conversion tool I used. It's removed now.